src/elvis_style.erl

```diff
@@ -304,9 +305,23 @@ result_node_line_col_fun(Msg) ->
 
 %% Rule checking
 
+-spec line_is_comment(binary()) -> true | false.
+line_is_comment(Line) ->
+    case re:run(Line, "^ *%") of
```

Comment:
I like that option, I would go with `skip_comments :: none | whole_line | any`
and don't forget to document your rule in [the wiki](https://github.com/inaka/elvis/wiki/Rules)